### PR TITLE
Added `bump-versions` to the readme

### PR DESCRIPTION
**Pull request by KevinMenden**
Added description of the `nf-core modules bump-versions` command to the readme.

## PR checklist

 - [x] This comment contains a description of changes (with reason)
 - [x] `CHANGELOG.md` is updated
 - [ ] If you've fixed a bug or added code that should be tested, add tests!
 - [ ] Documentation in `docs` is updated

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1102,6 +1107,49 @@ INFO     Linting module: star/align
...
+│ [!] 1  Module updated     │
+╰───────────────────────────╯
+╭─────────────────────────────────────────────────────────────╮
+│ Module name               │ Update message                  │     
+├───────────────────────────┤─────────────────────────────────┤
+│ bcftools/consensus        │ Module updated:  1.11 --> 1.12  │
+╰─────────────────────────────────────────────────────────────╯
+```
+
+If you don't want to update certain modules or want to update them to specific versions, you can make use of the `.nf-core.yml` configuration file. For example, you can precent the `star/align` module from being update by adding the following to the `.nf-core.yml` file:
```
Comment (ErikDanielsson) [inline README.md:1139]:
```suggestion
If you don't want to update certain modules or want to update them to specific versions, you can make use of the `.nf-core.yml` configuration file. For example, you can prevent the `star/align` module from being updated by adding the following to the `.nf-core.yml` file:
```

File: README.md [hunk 2]
```diff
@@ -1102,6 +1107,49 @@ INFO     Linting module: star/align
...
+```console
+$ nf-core modules bump-versions -d modules 
+
+                                          ,--./,-.
+          ___     __   __   __   ___     /,-._.--~\
+    |\ | |__  __ /  ` /  \ |__) |__         }  {
+    | \| |       \__, \__/ |  \ |___     \`-._,-`-,
+                                          `._,._,'
+
+    nf-core/tools version 1.15.dev0
```
Comment (ErikDanielsson) [inline README.md:1123]:
```suggestion
    nf-core/tools version 2.0.dev0
```
Not sure if this is the correct thing to do ...

**Review by ErikDanielsson — APPROVED** [review]
LGTM :+1:

File: README.md (hunk 2 above)
Comment (KevinMenden) [reply README.md:1123]: Yup that works, could actually even remove the dev tag here

**Pull request merged**